allure-report-face/src/plugins/report-stats/report-stats-widget/ReportStatsWidget.js

```diff
@@ -0,0 +1,25 @@
+import './styles.css';
+import {ItemView} from 'backbone.marionette';
+import template from './ReportStatsWidget.hbs';
+import ReportWidgetModel from './ReportWidgetModel';
+
+export default class ReportStatsWidget extends ItemView {
+    template = template;
+
+    initialize() {
+        this.model = new ReportWidgetModel();
+        this.model.fetch().then(
+            () => {
+                if (!this.isDestroyed) {
```

Comment:
report.json info is not in widgets.json, but i can't touch java to fix that because of baev rewrites this part. So when we will use new one, it will be replaced with right json.